packages/core/src/lib/constants.ts

```diff
@@ -1,4 +1,6 @@
 export const PACKAGE_NAME = "near-wallet-selector";
+export const LAST_SIGNED_IN_WALLET =
+  "near-wallet-selector:last_signed_in_wallet";
```

Comment:
I think we can remove the package name and the column `near-wallet-selector:` here otherwise it will be duplicated when you create a new `jsonStorage` :
```ts
  const jsonStorage = new JsonStorage(this.storage, PACKAGE_NAME);
```
It will look something like this in localstorage: 
`near-wallet-selector:near-wallet-selector:last_signed_in_wallet`

Would you also consider setting it as `lastSignedInWallet` instead of `last_signed_in_wallet` it will align with the way you set it in the store (state)?

```ts
export const LAST_SIGNED_IN_WALLET = "lastSignedInWallet";
```